components/api-monitoring/index.tsx

```diff
@@ -22,9 +19,6 @@ const getUptimeColor = (ratio: IRatio) => {
 };
 
 const getUptimeLabel = (ratio: IRatio) => {
-  if (!ratio.isActive) {
-    return 'Aucune donnée sur cette période.';
-  }
```

Comment:
So for instance we loose the concept of no data because the monitor did not exist yet.